docs/clients/p4rt-ctl.rst

```diff
@@ -91,6 +91,24 @@ services that are available for a p4runtime client.
 For information on security and enabling TLS on gRPC ports, refer to
 guides/security-guide.md
 
+Usage: p4rt-ctl [OPTIONS] COMMAND [ARG...]
+positional arguments:
+  command               Subcommand to run
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -g, --grpc_addr       GRPC_ADDR P4Runtime gRPC server address, format : <server IP>:<port>
+
+Details about supported option (``-g`` GRPC_ADDR)::
```

Comment:
```suggestion
Details about the supported option (``-g`` GRPC_ADDR)
```
The double colon at the end of the line is displayed in the text. I think this is because you have used in combination with the row of underlines, which makes this line a heading. One does not use a colon on a heading line. The colon serves to set off the following phrase or sentence in normal text. A heading automatically sets off the following text..